apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -253,6 +259,31 @@ router.post(
         }
       });
       return res.redirect(`/pl/jobSequence/${serverJob.jobSequenceId}`);
+    } else if (req.body.__action === 'send_grades') {
+      // Should something like this be in a model?
+      const assessment = await queryRow(
+        sql.select_assessment_in_course_instance,
+        {
+          unsafe_assessment_id: req.body.unsafe_assessment_id,
+          course_instance_id: res.locals.course_instance.id,
+        },
+        AssessmentSchema,
+      );
+      if (assessment === null) {
+        throw new Error('Invalid assessment.id');
+      }
+
+      serverJobOptions.description = 'LTI 1.3 send assessment grades to LMS';
+      const serverJob = await createServerJob(serverJobOptions);
+
+      serverJob.executeInBackground(async (job) => {
+        await updateLti13Scores(assessment.id, instance, job);
+
+        await queryAsync(sql.update_lti13_assessment_last_activity, {
+          assessment_id: assessment.id,
+        });
+      });
+      return res.redirect(`/pl/jobSequence/${serverJob.jobSequenceId}`);
```

Comment:
Surely this won't do what you want in production since this route is only mounted in dev mode:

https://github.com/PrairieLearn/PrairieLearn/blob/aa2ba37e44b236e67df44e6f5c9ae16811d58a6b/apps/prairielearn/src/server.js#L648-L655

Ditto for other redirects in this file.